native/machine-coding/password-generator/index.html

```diff
@@ -56,6 +56,11 @@
           <label for="symbols">Include Symbols</label>
         </div>
 
+        <div class="strength-container">
+          <p>Strength</p>
+          <div class="indicator" data-indicator></div>
```

Comment:
also add a tooltip on hover over the color if the password strength is 
**strong, weak or medium**